docs/basics/dataframe.md

```diff
@@ -203,13 +227,12 @@ on a series.
 Make a Python file with the following content:
 
 ```python exec="1" source="above" session="df_ex4"
-from typing import Any
-
 import narwhals as nw
+from narwhals.typing import DataFrameT
 
 
 @nw.narwhalify(eager_only=True)
-def func(df: nw.DataFrame[Any], s: nw.Series, col_name: str) -> int:
+def func(df: DataFrameT, s: nw.Series, col_name: str) -> int:
```

Comment:
should we be using a TypeVar when it's only used in one place? I think this one might be better staying as `nw.DataFrame[Any]`?